File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    const answer = []
    const word = s.split(' ')
    word.forEach((word) => {
        const arr = [...word].map((val, idx) => {
            return idx % 2 === 0 ? val.toUpperCase() : val.toLowerCase();
        })
        answer.push(arr.join(''))
    })
    return answer.join(' ')
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = []
    words = s.split(' ')
    for word in words:
        temp = ''
        for idx, s in enumerate(word):
            if idx % 2 == 0:
                temp += s.upper()
            else:
                temp += s.lower()
        answer.append(temp)
    return ' '.join(answer)